docs/contact.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+    <title>connect to SHA</title>
+	<link rel="stylesheet" href="css/contact.css">
+</head>
+<body>
+    <header class="header">
+        <div class="container">
+            <div class="header-content">
+                <h1 class=" header-title ">Get In Touch</h1>
+            </div>
+        </div>
+    </header>
+    <nav class="navbar sticky-top navbar-expand-lg navbar-light bg-white" data-spy="affix" data-offset-top="510">
+        <div class="container">
+            <button class="navbar-toggler ml-auto" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse mt-sm-20 navbar-collapse" id="navbarSupportedContent">
+                <ul class="navbar-nav mr-auto">
+                    <li class="nav-item">
+                        <button style="background-color: rgb(230, 159, 159); color: black;"> <a href="contact.html" class="nav-link"> <a href="index.html" class="nav-link">Home</a></button>  
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+    <div class="section contact" id="contact">
+        <div id="map" class="map"></div>
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-8">
+                    <div class="contact-form-card">
+                        <h4 class="contact-title">Mail To me</h4>
+                        <form>
+                            <div class="form-group">
+                                <input  class="form-control" type="text"  placeholder="First and Last Names" required class="textbox">
+                            </div>
+                            <div class="form-group">
+                                <input class="form-control" type="email" id="email" placeholder="email id" required>
+                            </div>
+                            <div class="form-group">
+                                <textarea class="form-control" placeholder=" Your Message"  required></textarea>
+                            </div>
+                            <div class="form-group ">
+                                <button style="background-color: red; color: black;">
+                                    <a href="mailto:nithinshanithu8@gmail.com" target="_blank" rel="noopener noreferrer" style="color: inherit; text-decoration: none; display: block; padding: 10px;">
```

Comment:
Instead of this link, using `action="mailto:nithinsha...."` as an attribute in the `<form>` element would enable the email to automatically populate with the data entered into the form.